aws-ecs/main.tf

```diff
@@ -59,13 +59,13 @@ resource "aws_ecs_task_definition" "controller" {
   family                   = "rookout_controller"
   requires_compatibilities = ["FARGATE"]
   network_mode             = "awsvpc"
-  cpu                      = 256
-  memory                   = 512
+  cpu                      = 4096 // 4CPU
```

Comment:
Why only 1GB memory?

Reply:
1. Because I'm matching the ones in `helm-charts`, it is important to have single value for all deployments
2. Because we didn't have any OOM errors for controllers